homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -1717,14 +1717,22 @@ class DeviceAudioQueue:
     """Flag to be set if audio samples were dropped because the queue was full."""
 
 
+@dataclass
```

Comment:
```suggestion
@dataclass(slots=True)
```

Not that it matters but possible since have py3.11 min